config/local_conda_mapping.yml.sample

```diff
@@ -1,2 +1,50 @@
 # See $GALAXY_ROOT/lib/galaxy/tools/deps/default_conda_mapping.yml for example mapping - 
 # additional site-specific mappings can be added to config/conda_mapping.yml.
+- from:
+    name: samtools
+    version: Null
+  to:
+    name: samtools
+    version: 1.3.1
+- from:
+    name: ucsc_tools
+    version: Null
+  to:
+    name: ucsc_tools
+    version: 332
+- from:
+    name: bedtools
+    version: Null
+  to:
+    name: bedtools
+    version: 2.26.0gx
+- from:
+    name: gnuplot
+    version: Null
+  to:
+    name: gnuplot
+    version: 4.6
+- from:
+    name: openbabel
+    version: 2.3.2
```

Comment:
This specifically because there is no openbabel 2.3.2 on conda.
In which case we might want to upgrade the datatype converters that use that version ?!
Fyi those are the tools that want openbabel 2.3.2:
```
cml_to_inchi_converter.xml
cml_to_mol2_converter.xml
cml_to_sdf_converter.xml
cml_to_smi_converter.xml
inchi_to_cml_converter.xml
inchi_to_mol2_converter.xml
inchi_to_mol_converter.xml
inchi_to_sdf_converter.xml
inchi_to_smi_converter.xml
mol2_to_cml_converter.xml
mol2_to_inchi_converter.xml
mol2_to_mol_converter.xml
mol2_to_sdf_converter.xml
mol2_to_smi_converter.xml
mol_to_cml_converter.xml
mol_to_inchi_converter.xml
mol_to_mol2_converter.xml
mol_to_smi_converter.xml
sdf_to_cml_converter.xml
sdf_to_inchi_converter.xml
sdf_to_mol2_converter.xml
sdf_to_smi_converter.xml
smi_to_cml_converter.xml
smi_to_inchi_converter.xml
smi_to_mol2_converter.xml
smi_to_mol_converter.xml
smi_to_sdf_converter.xml
smi_to_smi_converter.xml
```